src/xtopcom/xblockmaker/src/xtable_maker.cpp

```diff
@@ -282,7 +285,8 @@ xblock_ptr_t xtable_maker_t::leader_make_light_table(const xtableblock_proposal_
     return proposal_block;
 }
 
-xblock_ptr_t xtable_maker_t::backup_make_light_table(const xtableblock_proposal_input_t & proposal_input, const data::xblock_consensus_para_t & cs_para, xtablemaker_result_t & table_result) {
+xblock_ptr_t xtable_maker_t::backup_make_light_table(const xtablemaker_para_t & table_para, const data::xblock_consensus_para_t & cs_para, xtablemaker_result_t & table_result) {
```

Comment:
leader_make_light_table rename to make_table & backup_make_light_table rename to verify_table will be clean enough